public/app/features/transformers/docs/content.ts

```diff
@@ -321,14 +322,16 @@ export const transformationDocsContent: TransformationDocsContentType = {
   | Arizona   |    |           |            |            | 5      |
   | Arkansas  |    |           |            |            | 1      |
   | Somewhere |    |           |            |            | 5      |
+
+  This transformation enables you to augment your data by fetching additional information from external sources, providing a more comprehensive dataset for analysis and visualization.
```

Comment:
We avoid using the word "enable" in relation to people. In relation to features, it's okay.
```suggestion
  This transformation lets you augment your data by fetching additional information from external sources, providing a more comprehensive dataset for analysis and visualization.
```